src/shared.ts

```diff
@@ -2,6 +2,7 @@ export const proxyPath = '/_axiom';
 
 export const isBrowser = typeof window !== 'undefined';
 export const isEnvVarsSet = process.env.AXIOM_INGEST_ENDPOINT || process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT;
+export const isPrettyPrintEnabled = process.env.AXIOM_PRETTY_PRINT_ENABLED == 'false' ? false : true;
```

Comment:
Do you think using `NO_COLOR` instead would make more sense? https://no-color.org/